frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1338,6 +1361,101 @@ const AlertsPage_: React.FC<Alerts> = ({ data, loaded, loadError }) => {
     },
   ];
 
+  const severityRowFilter: RowFilter = {
```

Comment:
Prefer to keep this outside the components to avoid duplication (probably using `i18next.t`).

Reply:
i18next.t is iffy with row filters. It's causing missingKey errors that will break the CI tests.